README.md

```diff
@@ -19,12 +19,12 @@ The top-level folder of this repository is licensed under the MIT license.
 
 | Folder     | License | Description                                                         |
 | ---------- | --------| ------------------------------------------------------------------- |
-| DAISI      | GPLv2   | ns-3 network simulation setup for all components and applications   |
-| MINHTON    | MIT     | P2P tree-structured overlay network                                 |
+| daisi      | GPL-2.0 | ns-3 network simulation setup for all components and applications   |
```

Comment:
We could use the SPDX identifier for clarity: GPL-2.0-only

Reply:
I agree, but than we should change the header name from License to "SPDX Itendifier" or add another column...

Reply:
I agree, but than we should change the header name from License to "SPDX Itendifier" or add another column...

Reply:
Done, I updated the PR accordingly.